docs/user-guide/testing/test-execution-flow.md

```
@@ -0,0 +1,396 @@
+**仕様変更・機能追加時の対応**:
+
+- **新機能追加**: 既存のテストファイルに新しいテストケースを追加
+- **仕様変更**: 既存のテストケースを更新（TDDの原則に従って）
+- **機能削除**: 不要になったテストケースを削除
+
+**メリット**:
+- テスト実行時間が一定に保たれる
+- テストが常に最新の仕様を反映
+- メンテナンスが容易
+
+**phase-0、phase-1のような履歴管理はしない**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)